packages/forms-web-app/src/dynamic-forms/section.js

```diff
@@ -92,8 +93,15 @@ class Section {
 				requiredQuestionCount++;
 			}
 
-			// move to next question if answer not provided for this questio or for file upload questions the length of uploaded files is less than 1
-			if (!answer || answer.uploadedFiles?.length < 1) {
+			if (question.documentType && answer === 'yes') {
+				const relevantDocs = journeyResponse.answers.SubmissionDocumentUpload.filter(
+					(upload) => upload.type === question.documentType.name
+				);
+				missingRequiredFiles = relevantDocs.length < 1 ? true : false;
```

Comment:
Why not just do 
```
missingRequiredFiles = journeyResponse.answers.SubmissionDocumentUpload.some(
					(upload) => upload.type === question.documentType.name
				);
```

Reply:
I think it would need to be 
`!journeyResponse.answers.SubmissionDocumentUpload.some(
					(upload) => upload.type === question.documentType.name
				);`
				
as we want missingRequiredFiles to be true if there are none, but false if there are some relevant docs.